src/boot.tsx

```diff
@@ -31,7 +47,8 @@ export default (element: HTMLDivElement, hub: Hub) => {
     const exercise = JSON.parse(atob(decodeURIComponent(element.textContent)));
     settings.hint = exercise.hint;
     settings.language = exercise.language;
-    settings.pre_exercise_code = exercise.pre_exercise_code;
+    settings.lang_version = exercise.lang_version;
+    settings.pre_exercise_code = exercise.pre_exercise_code + getPackages(exercise.packages);
```

Comment:
I'm surprised your packages don't go before the pre exercise code.

Reply:
Duplicate https://github.com/datacamp/datacamp-light/pull/153#discussion_r302860272 😄